### ISSUE TITLE
SSH Server RHEL/ROCKY 8.6

### ISSUE THREAD

**Issue by DBasoco** [microsoft/vscode-remote-release#8240]
Hello I am using the Remote SSH extension version v0.98.0 and I am having issues installing the ssh server on a remote linux box. I have tried to set it up on both a RHEL 8.6 minimal install and a ROCKY 8.6 minimal install. My local box is a Windows 10 laptop. And the resolve error I receive is:

[13:12:39.257] Resolver error: Error: Failed to install the VS Code Server
	at m.ServerInstallError (c:\Users\David.Basoco\.vscode\extensions\ms-vscode-remote.remote-ssh-0.98.0\out\extension.js:1:584322)
	at f (c:\Users\David.Basoco\.vscode\extensions\ms-vscode-remote.remote-ssh-0.98.0\out\extension.js:1:578617)
	at t.handleInstallOutput (c:\Users\David.Basoco\.vscode\extensions\ms-vscode-remote.remote-ssh-0.98.0\out\extension.js:1:581404)
	at t.tryInstall (c:\Users\David.Basoco\.vscode\extensions\ms-vscode-remote.remote-ssh-0.98.0\out\extension.js:1:681793)
	at async c:\Users\David.Basoco\.vscode\extensions\ms-vscode-remote.remote-ssh-0.98.0\out\extension.js:1:644041
	at async t.withShowDetailsEvent (c:\Users\David.Basoco\.vscode\extensions\ms-vscode-remote.remote-ssh-0.98.0\out\extension.js:1:647359)
	at async t.resolve (c:\Users\David.Basoco\.vscode\extensions\ms-vscode-remote.remote-ssh-0.98.0\out\extension.js:1:645091)
	at async c:\Users\David.Basoco\.vscode\extensions\ms-vscode-remote.remote-ssh-0.98.0\out\extension.js:1:720839
[13:12:39.264] ------

I have successfully connected to another ROCKY 8.6 installation (from the same media as above) by using an rsync on the .vscode-server/bin folder from a CENTOS 7.9 installation I have. It returned this same error before I synced it up. I initially thought it was a bad sync, as that is a copy of the other server, but since I can replicate it on two fresh installs I think it is a different problem. Is Remote - SSH known to have issues on kernel 4.18.0-372.9.1.e18.x86_64?

**Comment by arozmarin** [comment]
I don't know will be same for you, but I had similar problem and I just install tar on rocky linux 9 and it worked.  I found in output tar could be found.

**Comment by eleanorjboyd** [comment]
Hello! Could you please provide the entire remote-ssh logs? The whole logs includes some settings and output which are helpful to the debugging process. You can find the logs by going to `view` in the menu bar, then selecting `output` and finally once the bottom window pops up select  `remote-ssh` from the dropdown on the right. Thanks!

**Comment by eleanorjboyd** [comment]
also I do not have specific experience with this problem but it seems like @arozmarin has found a fix for himself that could work. Thanks @arozmarin for your comment!

**Comment by eleanorjboyd** [comment]
Hello! In the logs it says `Server did not start successfully. Full server log at /home/kratos/.vscode-serve
> r/.ee2b180d582a7f601fa6ecfdad8d9fd269ab1884.log >>>`. Could you try and find that file on your remote and send it over? Thanks!

**Comment by DBasoco** [comment]
[kratos@kratos .vscode-server]$ cat .ee2b180d582a7f601fa6ecfdad8d9fd269ab1884.log
main: line 510: /home/kratos/.vscode-server/bin/ee2b180d582a7f601fa6ecfdad8d9fd269ab1884/bin/code-server: Permission denied

**Comment by eleanorjboyd** [comment]
A few questions, do you change the user once you connect? Can you check the umask value of that file to see the file access of the given user you are trying to connect with? Is this a new issue (regression) or is this your first time trying to use this extension? Thanks!

**Comment by DBasoco** [comment]
I can't switch users once I send the ssh connect command though vscode. And a top in a separate connection shows the kratos user id. All files have the correct operational octet and owner for kratos. This is a new issue. I believe it is in part to the paired down media we are using. I've used this extension in both the work environment and my home environment. We tried to use the extension on a full media install and it worked fine. But even after installing tar and python2.7 to be fully in accordance with documented requirements the error persists. Is there another requirement that is needed?

**Comment by eleanorjboyd** [comment]
what do you mean by `paired down media we are using`? I think your issue is the same as this [issue](https://github.com/microsoft/vscode-remote-release/issues/8060) so I am going to close this issue in favor of that issue. Please try the suggestions I had there about using the command "umask" to check file permissions

**Comment by DBasoco** [comment]
The iso that the system is made off of is a reduced size. I manually paired down the appstream repository to create media that can enter DoD environments. The umask is 0002 so it's open. From the other issue you referenced I checked that the offended partition is not mounted with noexec option. 

I have another system that runs centos 79 that the extension works perfect on. In order to make it operable on the rocky 86 image I need to rsync the /home/<user>/.vscode-server directory to the new system. It took several attempts and iterations to  get it to work but that isn't always repeatable and neither tar nor python2.7 are on that box.
